pages/your_domain/proteomics.md

```diff
@@ -56,10 +56,17 @@ For all steps within a FAIR proteomics data analysis pipeline software is needed
 
 
 ### Solutions
-- Within the proteomics community various converter software tools such as {% tool "proteowizard" %} were implemented, which support the conversion of mass spectrometer output formats to the mzML standard data format as well as other conversions to standard data formats.
-- Information on software tools that support HUPO-PSI standard data formats can be found on the standard format-specific web pages of the {% tool "proteomics-standards-initiative" %} (e.g., [mzML](https://www.psidev.info/mzML) , [mzIdentML](https://www.psidev.info/mzidentml) and [MZTAB](https://www.psidev.info/mztab)).
+- Within the proteomics community various converter software tools such as {% tool "msconvert" %} were implemented, which support the conversion of mass spectrometer output formats to the mzML standard data format as well as other conversions to standard data formats.
+- Information on software tools that support HUPO-PSI data formats can be found on the standard format-specific web pages of the HUPO-PSI (e.g., [mzML](https://www.psidev.info/mzML) , [mzIdentML](https://www.psidev.info/mzidentml) and [MZTAB](https://www.psidev.info/mztab) ).
 
 
+### Example of tools using PSI formats
```

Comment:
I was wondering if we could list these tools without making a new H3 subheading (since we like to stick with the description/considerations/solutions structure)

Reply:
Done in f3a3cd5